src/components/dex/Pair.vue

```diff
@@ -41,20 +42,30 @@ export default {
     },
 
     percentChangeAbsolute() {
-      return this.$store.state.tradeHistory ?
-        Math.abs(this.$store.state.tradeHistory.change24HourPercent) : 0;
+      return Math.round(((this.change24Hour)
+        / this.tickerData.open24hr) * 10000) / 100;
+    },
+
+    change24Hour() {
+      const tradeHistory = this.$store.state.tradeHistory;
+      const marketName = this.$store.state.currentMarket.marketName
+      return tradeHistory[marketName] ?
+        (tradeHistory[marketName].close24Hour - this.tickerData.open24hr) : 0;
```

Comment:
I think on Desktop we had to use BigNumber for both these values before the subtraction or we were getting some weird rounding issue.